projects/plugins/jetpack/src/class-deprecate.php

```diff
@@ -119,7 +183,11 @@ private function render_notice( $id, $text, $params = array() ) {
 			$params['additional_classes'][] = 'jetpack-deprecate-dismissible';
 		}
 
-		wp_admin_notice( $text, $params );
+		printf(
+			'<div id="%1$s" class="notice notice-warning is-dismissible jetpack-deprecate-dismissible" style="border-left-color: #000000;">%2$s</div>',
+			esc_html( $params['id'] ),
+			$text // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
```suggestion
			$text // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped Output already escaped in render_admin_notices
```

Reply:
Thanks, added in de5ae51d520b101582573b659068edb117df38d4